apkid/rules/dex/obfuscators.yara

```diff
@@ -69,7 +69,7 @@ rule dexguard : obfuscator
 rule dexprotector : obfuscator
 {
   meta:
-    description = "DexProtect"
+    description = "DexProtector"
```

Comment:
Please limit the scope of your changes to one "thing" per PR. Please move this change to a separate PR. Also, if you think this name should be changed, please explain why in that PR.